File: openresty/cdn-system/api/models/cert.go
package models

import (
	"time"
)

type Cert struct {
	ID          int        `json:"id" gorm:"primaryKey;autoIncrement"`
	UserID      int        `json:"uid" gorm:"column:uid"`
	Name        string     `json:"name"`
	Description string     `json:"des" gorm:"column:des"`
	Type        string     `json:"type"` // e.g., "upload", "lets"
	Domain      string     `json:"domain"`
	DNSAPI      int        `json:"dnsapi" gorm:"column:dnsapi"`
	Cert        string     `json:"cert"`
	Key         string     `json:"key"`
	StartTime   *time.Time `json:"start_time"`
	ExpireTime  *time.Time `json:"expire_time"`
	AutoRenew   bool       `json:"auto_renew"`
	CreateAt    time.Time  `json:"create_at"`
	UpdateAt    time.Time  `json:"update_at"`
	Enable      bool       `json:"enable"`
	TaskID      int64      `json:"task_id"`
	IssueTaskID int64      `json:"issue_task_id"`
	Version     int        `json:"version"`
}

func (Cert) TableName() string {
	return "cert"
}

File: cnn.net/src/Cnn.Common/Domain/Entities/Cert.cs
using SqlSugar;

namespace Cnn.Domain.Entities;

[SugarTable("cert")]
public class Cert
{
    [SugarColumn(IsPrimaryKey = true, IsIdentity = true)]
    public int Id { get; set; }

    public int? Uid { get; set; }

    public string? Name { get; set; }

    public string? Des { get; set; }

    public string? Type { get; set; }

    public string? Domain { get; set; }

    public int? Dnsapi { get; set; }

    [SugarColumn(ColumnName = "cert")]
    public string? CertPem { get; set; }

    public string? Key { get; set; }

    public DateTime? StartTime { get; set; }

    public DateTime? ExpireTime { get; set; }

    public bool? AutoRenew { get; set; }

    public DateTime? CreateAt { get; set; }

    public DateTime? UpdateAt { get; set; }

    public bool? Enable { get; set; }

    public long? TaskId { get; set; }

    public long? IssueTaskId { get; set; }

    public string? State { get; set; }

    public string? Ret { get; set; }

    public int? Version { get; set; }
}
